### PR TITLE
doc: done the correction

### PR DESCRIPTION
**Pull request by mariyasada**
done the correction and by mistake, i have closed the previous pull request. so do review it

### CODE REVIEW

File: component.css [hunk 1]
```diff
@@ -108,19 +108,19 @@
...
   {
-      font-size: 2rem;
-      list-style: none;
-      padding: 0.5rem;
-      font-weight: 500;
-      align-self: center;
-      /* margin-top:2rem; */
+    font-size: 1.8rem;
+    list-style: none;
+    padding: 0.3rem;
```
Comment (ShreyashMahajan) [inline component.css:120]: you can use px for less than 1rem

**Review by ShreyashMahajan — APPROVED** [review]
code looks good to merge

File: component.css [hunk 2]
```diff
@@ -217,10 +217,10 @@
 
     .sidebar-container{
        font-size: 1rem;
-       cursor: pointer;
        margin-left:0;
        padding: 0.3rem;
-       width: 40%;
+       /* width: 40%; */
```
Comment (JantuDeb) [inline component.css]: Remove commented code.

File: component.css [hunk 3]
```diff
@@ -108,19 +108,19 @@
...
       
   }
   .sidebar-list-item
   {
-      font-size: 2rem;
-      list-style: none;
-      padding: 0.5rem;
-      font-weight: 500;
-      align-self: center;
-      /* margin-top:2rem; */
```
Comment (JantuDeb) [inline component.css:123]: remove commented code

File: component.css (hunk 1 above)
Comment (JantuDeb) [reply component.css:120]: to be consistent with length we can use rem,

Comment (mariyasada) [reply component.css:120]: I have used px only in box-shadow property

**Review by JantuDeb — APPROVED** [review]
suggested some changes.

File: component.css (hunk 3 above)
Comment (mariyasada) [reply component.css:123]: okay

**Pull request merged**